databricks/provider.go

```diff
@@ -261,18 +263,24 @@ func providerConfigureAzureClient(d *schema.ResourceData, config *service.DBApiC
 
 // tryDatabricksCliConfigFile sets Host and Token from ~/.databrickscfg file if it exists
 func tryDatabricksCliConfigFile(d *schema.ResourceData, config *service.DBApiClientConfig) error {
-	configFile, err := homedir.Expand(d.Get("config_file").(string))
+	configFile := d.Get("config_file").(string)
+	configFilePath, err := homedir.Expand(configFile)
 	if err != nil {
-		return err
+		return nil
+	}
+	cfg, err := ini.Load(configFilePath)
+	if err != nil && configFile == configFileDefault {
```

Comment:
@EliiseS the this change ignores loading data from `~/.databrickscfg`, so i won't agree with it. 

But i totally agree that `azure_auth` in provider must take precedence

Reply:
The problem is that it's valid for no configuration to be set, due to the fact the configuration is called multiple times turning the plan phase, sometimes with no config set yet. My educated guess for why that happens is because f the provisioning of the workspace. 

How would you like me to handle this?